homeassistant/components/daikin/climate.py

```diff
@@ -54,6 +54,21 @@
     ATTR_TARGET_TEMPERATURE: 'stemp'
 }
 
+HA_FAN_MODES_2_DAIKIN = {
+    '1': '1',
+    '2': '2',
+    '3': '3',
+    '4': '4',
+    '5': '5',
+    'Auto': 'auto',
+    'Silence': 'silence',
+    'Low': 'low',
+    'Medium': 'mid',
+    'High': 'high',
+}
+
+DAIKIN_FAN_MODES_2_HA = {v: k for k, v in HA_FAN_MODES_2_DAIKIN.items()}
```

Comment:
Where is this used?

Reply:
Oh, that is the stuff to only support HA modes.. I'll remove that, sorry.